AdobeMediaGalleryApi/Model/Asset/Command/GetByIdInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGalleryApi\Model\Asset\Command;
+
+/**
+ * Get media asset by id.
+ * @api
+ */
+interface GetByIdInterface
+{
+    /**
+     * Get media asset by id.
+     *
+     * @param int $assetId
+     *
+     * @return \Magento\AdobeMediaGalleryApi\Api\Data\AssetInterface
+     * @throws \Zend_Db_Statement_Exception
```

Comment:
It's better to catch all Zend Exceptions in the implementation and don't expose them to the API level. Also, it adds an additional dependency to ZF1 to the API.